dist/scss/50_objects/_o-footer.scss

```
@@ -1,3 +1,6 @@
+@import "../10_settings//spacing_units";
```

we've generally just been importing all settings when they are needed. This way is fine but lets try and keep everything consistent

```suggestion
@import '../10_settings/__all';
```